### PR TITLE
support for setting report directory

### PR DESCRIPTION
**Pull request by marcozwyssig**

### CODE REVIEW

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy [hunk 1]
```diff
@@ -49,7 +50,12 @@ class NUnit extends ConventionTask {
     }
 
     File getReportsFolder() {
-        new File(outputFolder, 'reports')
+        if( customReportFolder == null)
```
Comment (gluck) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:53]: can you use a convention instead ?

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy [hunk 2]
```diff
@@ -19,7 +19,8 @@ class NUnit extends ConventionTask {
     def timeout
     def runList
     def run
-    boolean useX86 = false
+	String customReportFolder
```
Comment (gluck) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:22]: indent

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy (hunk 1 above)
Comment (marcozwyssig) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:53]: Hi Francois,

I’m not so experienced in developing gradle pluings. Therefore I’m not sure what you mean with using a convention. Maybe you give me a hint what exactly you mean.

Thanks

Regards,
Marco

File: src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy (hunk 2 above)
Comment (marcozwyssig) [inline src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy:22]: Dear Francois,

I haven’t seen the wrong indent. I have to check it again.

Regards,
Marco

**Comment by gluck** [comment]
I implemented this in master, let me know if that works for you

**Comment by marcozwyssig** [comment]
Sure. Let me know when it is ready, thus I can test it.

Von meinem iPhone gesendet

Am 23.11.2015 um 13:41 schrieb Francois Valdy <notifications@github.com<mailto:notifications@github.com>>:

I implemented this in master, let me know if that works for you

##

**Pull request closed without merging**